### ISSUE TITLE
How to do Deeplink testing using EarlGrey?

### ISSUE THREAD

**Issue by satadip1994** [google/EarlGrey#790]
Is it possible to do deeplink testing using EarlGrey?

**Comment by tirodkar** [comment]
Yeah, you can move in between applications using methods provided here:

https://github.com/google/EarlGrey/blob/earlgrey2/TestLib/EarlGreyImpl/EarlGreyImpl%2BXCUIApplication.h

Use XCUITest to interact with the other application.

**Comment by hiteshjain4** [comment]
@tirodkar  : Is it possible to achieve this with earlgrey 1.15 version ?
